dll/win32/uxtheme/buffer.c

```diff
@@ -41,35 +63,107 @@ HRESULT WINAPI BufferedPaintUnInit(VOID)
 /***********************************************************************
  *      BeginBufferedPaint                                 (UXTHEME.@)
  */
-HPAINTBUFFER WINAPI BeginBufferedPaint(HDC hdcTarget,
-                                       const RECT * prcTarget,
-                                       BP_BUFFERFORMAT dwFormat,
-                                       BP_PAINTPARAMS *pPaintParams,
-                                       HDC *phdc)
+HPAINTBUFFER WINAPI BeginBufferedPaint(HDC targetdc, const RECT *rect,
+        BP_BUFFERFORMAT format, BP_PAINTPARAMS *params, HDC *retdc)
 {
-    static int i;
+#if (defined(_MSC_VER))
+    RGBQUAD rgb;
+    //TODO: Fix this for MSVC
+    __debugbreak();
```

Comment:
is this what Wine had?

Reply:
oh this is leftover from working fixing build ,
@Splitwirez  You're going to have to spend a little time finding a genuine fix to this and putting __REACTOS__ brackets around it like I do here